content/en/tracing/guide/_index.md

```diff
@@ -82,4 +82,5 @@ These tutorials walk you through setting up a sample multi-service application w
     {{< nextlink href="/tracing/guide/setting_up_apm_with_kubernetes_service/" >}}Setting up APM with Kubernetes Service{{< /nextlink >}}
     {{< nextlink href="/tracing/guide/aws_payload_tagging/" >}}Capture requests and responses from AWS services automatically{{< /nextlink >}}
     {{< nextlink href="/tracing/guide/service_overrides/" >}}Service Overrides{{< /nextlink >}}
+    {{< nextlink href="/tracing/guide/code_origins/" >}}Capture Code Origins for spans{{< /nextlink >}}
```

Comment:
I'm not sure about the "Capture" here.  Makes more sense, to me, to just have "Code Origins for spans."  And maybe capitalize "spans" here if this is supposed to be a section title.

Reply:
```suggestion
    {{< nextlink href="/tracing/guide/code_origins/" >}}Code Origins for Spans{{< /nextlink >}}
```